app/styles/global.css

```diff
@@ -1,6 +1,7 @@
 /* This is a global CSS file  */
 @import url('react-big-calendar/lib/css/react-big-calendar.css');
 @import url('react-big-calendar/lib/addons/dragAndDrop/styles.css');
+@import url('react-toastify/dist/ReactToastify.css');
 
```

Comment:
can we move the `rbc` styles to `index.tsx` route , something lke
`import calStyles from 'react-big-calendar/lib/css/react-big-calendar.css';`
`
export const links: LinksFunction = () => [
  { rel: 'stylesheet', href: styles },
  { rel: 'stylesheet', href: calStyles },
];`

Reply:
reference:https://remix.run/docs/en/main/guides/styling#regular-stylesheets

Reply:
@isVivek99 that is for cases where we need a css file for a single route. We need all these css files in many other routes in the project and it will not be efficient to repeat this step in each route.

Reply:
ok, i thought we are using RBC only on index route